dnsmenace.py

```
@@ -1,67 +1,514 @@
+            "--domain", "-d",
+            help="Domain name to query (FQDN)",
+            prompt="Enter domain to query",
+        ),
+    ],
+    record_types: Annotated[
+        list[RecordType],
+        typer.Option(
+            "--type", "-t",
+            help="DNS record type(s) to query",
+        ),
+    ] = [RecordType.A],
```

Ruff (0.14.14)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)